modules/programs/micro/bindings.nix

```
@@ -0,0 +1,13 @@
+lib:
+lib.mkOption {
+  type = lib.types.attrsOf lib.types.str;
+  default = { };
+  example = {
+    "Alt-/" = "lua:comment.comment";
+    "CtrlUnderscore" = "lua:comment.comment";
+  };
+  description = ''
+    Rebind keys and key combinations to certain actions.
+    See [here](https://github.com/zyedidia/micro/blob/master/runtime/help/keybindings.md) for further information.
```

Did you check the generated documentation. I don't think there is anything that detects whether the description is using markdown or docbook. Perhaps safer to use docbook?
```suggestion
    See <link xlink:href="https://github.com/zyedidia/micro/blob/master/runtime/help/keybindings.md"/>
    for further information.
```